template/nuxt-app/assets/vars/colors.json

```diff
@@ -3,6 +3,7 @@
 	"primary-color-dark": "#008c44",
 	"secondary-color": "#e8179e",
 	"offwhite": "#edf0f5",
+	"grey": "#cccccc",
```

Comment:
What if we rename to `disabled-fill` or `disabled-bkgd` ... something like that

Reply:
yea I like `disabled-fill`. I changed it.